libminifi/test/unit/StringUtilsTests.cpp

```diff
@@ -17,32 +17,36 @@
  */
 
 #include <string>
+#include <list>
+#include <set>
 #include <vector>
 #include <cstdlib>
 #include "../TestBase.h"
 #include "core/Core.h"
 #include "utils/StringUtils.h"
 
-using org::apache::nifi::minifi::utils::StringUtils;
+using org::apache::nifi::minifi::utils::StringUtils::split;
+using org::apache::nifi::minifi::utils::StringUtils::replaceEnvironmentVariables;
+using org::apache::nifi::minifi::utils::StringUtils::join;
 
 TEST_CASE("TestStringUtils::split", "[test split no delimiter]") {
   std::vector<std::string> expected = { "hello" };
-  REQUIRE(expected == StringUtils::split("hello", ","));
+  REQUIRE(expected == split("hello", ","));
 }
 
 TEST_CASE("TestStringUtils::split2", "[test split single delimiter]") {
   std::vector<std::string> expected = { "hello", "world" };
-  REQUIRE(expected == StringUtils::split("hello world", " "));
+  REQUIRE(expected == split("hello world", " "));
 }
 
 TEST_CASE("TestStringUtils::split3", "[test split multiple delimiter]") {
   std::vector<std::string> expected = { "hello", "world", "I'm", "a", "unit", "test" };
-  REQUIRE(expected == StringUtils::split("hello world I'm a unit test", " "));
+  REQUIRE(expected == split("hello world I'm a unit test", " "));
 }
 
 TEST_CASE("TestStringUtils::split4", "[test split classname]") {
   std::vector<std::string> expected = { "org", "apache", "nifi", "minifi", "utils", "StringUtils" };
-  REQUIRE(expected == StringUtils::split(org::apache::nifi::minifi::core::getClassName<org::apache::nifi::minifi::utils::StringUtils>(), "::"));
+  REQUIRE(expected == split("org::apache::nifi::minifi::utils::StringUtils", "::"));
```

Comment:
This allows us to inline test for a separate function, so the change seemed unnecessary.

Reply:
That's true, but it basically it wasn't a nice idea to add a bunch of util function to a class instead of a namespace. 

I agree that this is a useful test, I will find another class and use split on the name of that.